docs/ingestion/native-batch.md

```diff
@@ -1133,8 +1133,14 @@ the [S3 input source](#s3-input-source) or the [Google Cloud Storage input sourc
 
 ### HTTP Input Source
 
-The HTTP input source is to support reading files directly
-from remote sites via HTTP.
+The HTTP input source is to support reading files directly from remote sites via HTTP.
+
+> **NOTE:** The Druid Console runs with the same privileges as the operating system account that runs the Druid processes, for example the Indexer, Middle Manager, and Peon. This means the `HTTPInputSource` has access to data at any location where the Druid process has permissions. For example, using `HTTPInputSource`, a console user has access to internal network locations where the they would be denied access otherwise.
+
+> **WARNING:** `HTTPInputSource` is not limited to the HTTP or HTTPS protocols. It uses the Java `URI` class that supports HTTP, HTTPS, FTP, file, and jar protocols by default. This means you should never run Druid under the `root` account, because a user can use the file protocol to access any files on the local disk.
+
+For more information about security best practices, see [Security overview](..operations/security-overview.md).
```

Comment:
typo?
```suggestion
For more information about security best practices, see [Security overview](../operations/security-overview.md).
```

Reply:
1 question and a comment:

1. Should we link to the best practices section instead of just the entire page?
2. Given these notes and warnings about how ingestion works, I think there should be a bullet point under the best practices section that clearly states, only trusted users should be granted WRITE permissions to a DATASOURCE